Module4_Labs/Lab4_Maze_Solver_With_Stack/Cards/21.md

```diff
@@ -6,3 +6,5 @@
 
```

Comment:
The badges html comment has an extra >, which is showing up. Delete it so it doesn't show in the actual card